test/functional/wallet_labels.py

```diff
@@ -28,6 +28,44 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def invalid_label_name_test(self):
+        node = self.nodes[0]
+        address = node.getnewaddress()
+        pubkey = node.getaddressinfo(address)['pubkey']
+        rpc_calls = [
+            [node.getnewaddress],
+            [node.setlabel, address],
+            [node.getaddressesbylabel],
+            [node.importpubkey, pubkey],
```

Comment:
In 65e78bda7cd23ad7b6ede63c6bf5ffe6f552c71d "test: Invalid label name coverage"

`importpubkey` is a legacy wallet only RPC. The reason this test still passes with `--descriptors` is because the test framework has this (and some of the other import RPCs) aliased to `importdescriptors`.

Reply:
Considering this PR has 4 ACKs is it worth the change? The test is executed with both `--legacy-wallet` and `--descriptors` anyway.